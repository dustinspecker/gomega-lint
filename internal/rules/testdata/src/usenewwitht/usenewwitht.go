package usenewwitht

import (
	"testing"

	"github.com/onsi/gomega"
	. "github.com/onsi/gomega"

	"customasserter"
)

func main() {
	t := &testing.T{}

	gomega.NewGomegaWithT(t) // want `NewGomegaWithT is deprecated. Use NewWithT instead.`
	NewGomegaWithT(t)        // want `NewGomegaWithT is deprecated. Use NewWithT instead.`

	gomega.NewWithT(t)
	NewWithT(t)

	customasserter.NewGomegaWithT(t)
}
