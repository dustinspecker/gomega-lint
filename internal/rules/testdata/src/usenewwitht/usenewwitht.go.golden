package usenewwitht

import (
	"testing"

	"github.com/onsi/gomega"
	. "github.com/onsi/gomega"

	"customasserter"
)

func main() {
	t := &testing.T{}

	gomega.NewWithT(t) // want `NewGomegaWithT is deprecated. Use NewWithT instead.`
	NewWithT(t)        // want `NewGomegaWithT is deprecated. Use NewWithT instead.`

	gomega.NewWithT(t)
	NewWithT(t)

	customasserter.NewGomegaWithT(t)
}
